File: hackerrank/easy/0009-a-very-big-sum/python/solution.py
## solution.py

from typing import List

def a_very_big_sum(ar: List[int]) -> int:
    return sum(ar)


# --- demo run ---
if __name__ == "__main__":
    arr = [1000000001, 1000000002, 1000000003, 1000000004, 1000000005]
    print(a_very_big_sum(arr))  # 5000000015

File: hackerrank/easy/0009-a-very-big-sum/swift/solution.swift
// solution.swift

func aVeryBigSum(ar: [Int]) -> Int {
    return ar.reduce(0, +)
}

// --- demo run ---
let arr = [1_000_000_001, 1_000_000_002, 1_000_000_003, 1_000_000_004, 1_000_000_005]
print(aVeryBigSum(ar: arr))  // 5000000015